### ISSUE TITLE
Add visualisations for all spinners

### ISSUE THREAD

**Issue by curio-sitas** [AshlinHarris/Spinners.jl#51]
As more and more spinners with their own identifier will be added, it may be appropriate to create an automated (CI at build time ?) visual gallery showing each spinner associated with its identifier; It is easier than going through code to find the spinner of interest as well as testing dozens of them.

**Comment by AshlinHarris** [comment]
Agreed!

I made the examples on the README manually with a screen capture, which is fine for a short curated list. Generating a complete list would be great for users, as well as for testing.

I guess something like REPLay.jl is best for automating this. I haven't used any tool like that, and I'm hoping there's a way to disable the typing effect for the input command.

**Comment by AshlinHarris** [comment]
The [Replay.jl](https://github.com/AtelierArith/Replay.jl) README recommends using asciinema for generating animations from terminal output:
```
$ pip3 install asciinema # install `asciinema`
$ asciinema rec output.cast --command "julia examples/helloworld/app.jl"
$ asciinema play output.cast
```

**Comment by curio-sitas** [comment]
> The [Replay.jl](https://github.com/AtelierArith/Replay.jl) README recommends using asciinema for generating animations from terminal output:
> 
> ```
> $ pip3 install asciinema # install `asciinema`
> $ asciinema rec output.cast --command "julia examples/helloworld/app.jl"
> $ asciinema play output.cast
> ```

Yeah i saw their examples, but i searched up for a more julian approach without call other tools. But i ended up to the conclusion that there is any package capable of drawing utf emojis... 

But we could add the Replay.jl way of doing it in the CI, i'll do some tests

**Comment by curio-sitas** [comment]
I manage to use [Replay.jl](https://github.com/AtelierArith/Replay.jl)

I observed that : 

* It does not work on Windows (using a debian VM on wsl works fine)
* Using [asciinema](https://github.com/asciinema/asciinema) (**python**) provides a `.cast` file and not a `GIF` file
* We can create GIFs via [agg](https://github.com/asciinema/agg) (**Rust**) or using the asciinema API (upload) to create an online video

I will search a way to avoid printing the julia header as well as the package loading command line.

I will create a sub-project in the repo and a custom Github workflow to loop over all the spinner definitions.
Note that this has to be run during CI in order to use julia, python and rust to generate gifs. In the future, the documentation should print these gifs after they have been generated.

Gif result after using `julia+asciinema+agg` : 

![test](https://user-images.githubusercontent.com/26415466/197365867-636dcaa6-845c-4724-8fd3-40f86aac2023.gif)

**Comment by curio-sitas** [comment]
Asciinema works fine ! As we can see on the uploaded version : 
[![asciicast](https://asciinema.org/a/3d6WVpcr8MESiN32yOJ7DXASK.svg)](https://asciinema.org/a/3d6WVpcr8MESiN32yOJ7DXASK)

But `agg` does not manage to print emojis... More research to do ! This is related to the fonts used to generate de image.

![test](https://user-images.githubusercontent.com/26415466/197394419-3c65c905-2aeb-4097-ab50-7b181ff790a4.gif)

**Comment by curio-sitas** [comment]
Tried on a native Unix machine (Linux Manjaro) but presented the same problem as on WIndows. The `agg`program do not manage to draw emojis (even with a fallback font). One should consider to use asciinema's online API to generate the visuals
